examples/complete/main.tf

```diff
@@ -0,0 +1,56 @@
+# Lookup data from given enterprise account
+
+data "ibm_enterprises" "enterprise" {
+  name = var.enterprise_name
+}
+
+# Call root level module to create a hierarchy of enterprise child accounts and account groups
+module "enterprise" {
+  source                            = "../.."
+  enterprise_crn                    = data.ibm_enterprises.enterprise.enterprises[0].crn
+  enterprise_primary_contact_iam_id = data.ibm_enterprises.enterprise.enterprises[0].primary_contact_iam_id
+  enterprise_account_groups = [
+    {
+      key_name        = "${var.prefix}-group-key-1"
+      name            = "${var.prefix}_account_group_1"
+      parent_key_name = null
+    }
+  ]
+  enterprise_accounts = [
+    {
+      key_name               = "${var.prefix}-acc-key-1"
+      name                   = "${var.prefix}_account_1"
+      parent_key_name        = null
+      add_owner_iam_policies = true # this field enable child account to have IAM_APIKey with owner IAM policies
+    }
+  ]
+}
+
+#Create a new SM instance if not using an existing one
+module "secrets_manager" {
+  source               = "terraform-ibm-modules/secrets-manager/ibm"
+  version              = "1.17.8"
```

Comment:
can you update the example so it supports passing an existing secrets manager instance like I mentioned on our call? You can use [this](https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/blob/main/examples/ha-complete/main.tf) as a reference. 
And then in the test you can pass our existing SM instance so it wont create a new one (like [this](https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/blob/4b715db74c0c777296d572ffc40d6837e1a30fdf/tests/pr_test.go#L59-L60))